code/modules/mod/mod_theme.dm

```diff
@@ -289,7 +289,7 @@
 		offering full view of the land and its soon-to-be-dead inhabitants. The armor plating has been trimmed down to \
 		the bare essentials, geared far more for environmental hazards than combat against fauna; however, \
 		this gives way to incredible protection against corrosives and thermal protection good enough for \
-		both casual backstroking through molten magma and romantic walks through arctic terrain. \
+		traversing the hostile climate of scorching hot barren planets, molten, and volcanic worlds like Epsilon Eridanii II. \
```

Comment:
Keep your PR atomic please, its required by our github contributor guidelines

Reply:
![Discord_B0NecXzrAk](https://github.com/ParadiseSS13/Paradise/assets/62493359/5e70cb64-c04e-471a-bced-e12df7cde9fc)
Qwerty told me to do it here lol